Update mutation-verifier.md (#64)
Certora/Documentation: docs/gambit/gambit.md

In addition to the mutated source files, Gambit also produces a JSON report of the mutants produced,
which can be found in `out/results.json`.
The source is [publicly available](https://github.com/Certora/gambit).

> **Note that we currently only support MacOS and Linux. We have not tested on Windows machines. **

Reviewer: ```suggestion
**Note that we currently only support MacOS and Linux. Windows platform is currently unsupported. **
```

>     you provide Solidity's [allowed paths][allowed] to `solc` using the `--allow-paths` argument.

Reviewer 2: Provide Solidity's [allowed paths][allowed] to `solc` using the `--allow-paths` argument.

>     For example:

Reviewer 2: _Example:_
